---
title: 'Codonphyml: Fast maximum likelihood phylogeny estimation under codon substitution
  models'
date: '2013-01-01'
volume: '30'
number: '6'
pages: 1270-1280
publishDate: '2021-02-08T11:56:33.471520Z'
authors:
- Marcelo Serrano Zanetti
- Manuel Gil
- Stefan Zoller
- Maria Anisimova
abstract: Recently, Markov models of codon substitution have come into the spotlight.
  By incorporating the structure of the genetic code and the selection intensity at
  the protein level they provide a more realistic representation of protein-coding
  sequences than nucleotide or amino acid models. Thus, for protein-coding genes phylogenetic
  inference is expected to be more accurate under codon models. So far, phylogeny
  reconstruction under codon models has been elusive due to computational difficulties
  of dealing with high dimension matrices. Here we present a fast maximum likelihood
  package for phylogenetic inference, CodonPhyML offering hundreds of different codon
  models, the largest variety to date, for phylogeny inference by maximum likelihood.
  CodonPhyML is tested on simulated and real data, and is shown to offer excellent
  speed and convergence properties. In addition, CodonPhyML includes most recent fast
  methods for estimating phylogenetic branch supports, and provides an integral framework
  for models selection, including amino acid and DNA models.
publication: Molecular Biology and Evolution
url_pdf: http://mbe.oxfordjournals.org/content/30/6/1270.abstract
doi: 10.1093/molbev/mst034
featured: false
sg-areas:
---
